src/index.js

```diff
@@ -208,8 +193,9 @@ class HCaptcha extends React.Component {
     }
 
     render () {
+      // Keep elementId for backwards compatibility
       const { elementId } = this.state;
-      return <div id={elementId}></div>
+      return <div ref={this.myRef} id={elementId}></div>
```

Comment:
Could we remove `elementId` then? Doesn't seem like its needed anymore.

Reply:
I chose to keep it to be backwards compatible.